.idea/misc.xml

```diff
@@ -1,13 +1,16 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
+  <component name="JavaScriptSettings">
```

Comment:
It would probably makes sense to remove the IDEA specific files from the repository.

Reply:
I remove the file in latest commit.